var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -169,6 +170,22 @@ def install(self, spec, prefix):
         except OSError:
             pass
 
+        # Environment views do not currently support symlinks.
+        # Since the include/ and lib64/ directories in cuda are
+        # symlinks to subdirectories of targets/ and thus do
+        # not appear in the view, we manually replace them with
+        # copies of the directories they point to.
+        # For reference, see https://github.com/spack/spack/issues/19531
+        include_dir = join_path(prefix, 'include')
+        pointing_to = join_path(prefix, os.readlink(include_dir))
+        os.remove(include_dir)
+        shutil.copytree(pointing_to, include_dir)
```

Comment:
@alalazo  Don't we usually advocate use of Spack's `copy_tree` (https://spack.readthedocs.io/en/latest/llnl.util.html?highlight=copy_tree#llnl.util.filesystem.copy_tree) instead of using `shut.copytree`?

Reply:
Not sure, maybe @adamjstewart has an opinion on this.

Reply:
Yes, it used to be a wrapper around `shutil.copytree` but we added additional logic that better handles symlinks.